### PR TITLE
mercury root and integrations documentation

### PR DESCRIPTION
**Pull request by IlonaShishov**
Signed-off-by: Ilona Shishov <Ilona.Shishov@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ The provided reference implementation can serve as an example. While some result
...
 
+## Components
+
+### Integrations
+
+The Integrations component generates routes between the different components of each Service Domain in order to promote the free flow of data across the system.
+
+### Mercury Framework
+
+The Mercury Framework contains the core structure of all Service Domains and their implementations of behavior as well as a collection of libraries and tools to facilitate the integration between them.
```
Comment (ruromero) [inline README.md:22]: Add a link to the readme

File: README.md [hunk 2]
```diff
@@ -11,6 +11,20 @@ The provided reference implementation can serve as an example. While some result
...
+
+### Mercury Operator
```
Comment (ruromero) [inline README.md:24]: Add a link to the readme

File: README.md [hunk 3]
```diff
@@ -11,6 +11,20 @@ The provided reference implementation can serve as an example. While some result
 
 We invite the collaboration of technology experts and experienced business practitioners across the financial services industry to join and participate in co-creation.
 
+## Components
+
+### Integrations
```
Comment (ruromero) [inline README.md:16]: Add a link to the readme

File: README.md (hunk 1 above)
Comment (ruromero) [reply README.md:22]: I would add that this is the starting point for developers willing to implement any given service domain

File: README.md [hunk 4]
```diff
@@ -11,6 +11,20 @@ The provided reference implementation can serve as an example. While some result
...
+The Integrations component generates routes between the different components of each Service Domain in order to promote the free flow of data across the system.
+
+### Mercury Framework
+
+The Mercury Framework contains the core structure of all Service Domains and their implementations of behavior as well as a collection of libraries and tools to facilitate the integration between them.
+
+### Mercury Operator
+
+The Mercury Operator manages the deployment of the Service Domain Clusters by creating infrastructure components essential for service domains to function and communicate.
+
```
Comment (ruromero) [inline README.md:27]: I'd add that this is the starting point for cluster administrators willing to deploy an ecosystem of Service Domains on their application cluster

**Review by ruromero — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IlonaShishov) [reply README.md:22]: done, thank you

Comment (IlonaShishov) [reply README.md:22]: done, thank you

File: README.md (hunk 4 above)
Comment (IlonaShishov) [reply README.md:29]: done, thank you

Comment (IlonaShishov) [reply README.md:29]: done, thank you

File: README.md (hunk 2 above)
Comment (IlonaShishov) [reply README.md]: done, thank you

File: README.md (hunk 3 above)
Comment (IlonaShishov) [reply README.md]: done, thank you

File: mercury-framework/examples/README.md [hunk 5]
```diff
@@ -30,9 +30,26 @@ The customer credit rating service domain always returns the same value for any
 1. Create a Minikube cluster with the Mercury Operator and all the dependencies. 
 Look at the [Mercury Operator installation instructions](../../mercury-operator/README.md)
 2. Create a Service Domain Cluster
+
+   `minikube start -p service-domain`
+
 3. Create the different Service Domains (Customer Offer, Party Routing Profile and Customer
 Credit Rating)
+
+   `kubectl apply -f deploy/01_deployment.yaml`
```
Comment (ruromero) [inline mercury-framework/examples/README.md:39]: This resource should be replaced with the custom resources that the operator will use. ServiceDomainCluster and ServiceDomains (one for each)

File: mercury-framework/examples/README.md [hunk 6]
```diff
@@ -30,9 +30,26 @@ The customer credit rating service domain always returns the same value for any
...
+
 4. Initiate a Customer Offer
+   
+    ``curl -v -H "Content-Type: application/json" `minikube service customer-offer-camelk-rest --url=true`/customer-offer/${SD_REFERENCE}/customer-offer-procedure/initiation -d '{"procedure": {"customerReference": "${CUSTOMER_REFERENCE}"}}``
```
Comment (ruromero) [inline mercury-framework/examples/README.md:43]: The integrations and kubernetes services will be created by the operator, check the name of the service

File: mercury-framework/examples/README.md [hunk 7]
```diff
@@ -30,9 +30,26 @@ The customer credit rating service domain always returns the same value for any
...
+
 5. Check the Party Routing Profile exists for this Customer Reference
+    * For specific Customer Reference (CUSTOMER_REFERENCE)
+   
+      ``curl -v `minikube service party-routing-profile-camelk-rest --url=true`/party-routing-profile/${SD_REFERENCE}/party-state/${CUSTOMER_REFERENCE}/status/${BQ_REFERENCE}``
```
Comment (ruromero) [inline mercury-framework/examples/README.md:48]: check the name of the service created by the operator

File: mercury-framework/examples/README.md [hunk 8]
```diff
@@ -30,9 +30,26 @@ The customer credit rating service domain always returns the same value for any
...
+   
+    * For all Customer References
+   
+      ``curl -v `minikube service party-routing-profile-camelk-rest --url=true`/party-routing-profile/&{SD_REFERENCE}/party-state``
```
Comment (ruromero) [inline mercury-framework/examples/README.md:52]: same

**Review by ruromero — CHANGES_REQUESTED** [review]

**Review by ruromero — APPROVED** [review]

**Pull request merged**